CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Current release (in development)
 
+## 0.3.0
```

Comment:
This saves me time in cutting the release (which I planned on doing for this change so it can get out there), but in the future, let's add things to the current release section rather than targeting a specific release number that doesn't exist yet.

Reply:
That makes sense. Updated